lib/log_monitor.rb

```diff
@@ -59,27 +74,21 @@ def check_fails(log_entry)
     end
 
     unless logstash_tags.empty?
-      logstash_sender.log(log_entry, logstash_tags)
+      logstash_sender.log(log_entry, logstash_tags, last_success)
     end
```

Comment:
Could single-line these last two boolean checks to clean this method up a little, e.g. - 

`logstash_tags << "cdn_fallback" if cdn_fall_back?(log_entry)`